core/primitives-core/src/version.rs

```diff
@@ -219,24 +219,26 @@ impl ProtocolFeature {
             ProtocolFeature::DecreaseFunctionCallBaseCost => 66,
             ProtocolFeature::YieldExecution => 67,
 
+            // Congestion control should be enabled BEFORE stateless validation, so it has a lower version.
+            ProtocolFeature::CongestionControl => 80,
+
+            // Stateless validation features.
+            ProtocolFeature::StatelessValidationV0
+            | ProtocolFeature::LowerValidatorKickoutPercentForDebugging
+            | ProtocolFeature::SingleShardTracking
+            | ProtocolFeature::StateWitnessSizeLimit
+            | ProtocolFeature::PerReceiptHardStorageProofLimit
+            | ProtocolFeature::PartialEncodedStateWitness
+            | ProtocolFeature::WitnessTransactionLimits
+            | ProtocolFeature::OutgoingReceiptsSizeLimit
+            | ProtocolFeature::NoChunkOnlyProducers
+            | ProtocolFeature::ChangePartialWitnessDataPartsRequired
+            | ProtocolFeature::BiggerCombinedTransactionLimit => 81,
```

Comment:
No need to do it now but we can take it further and replace all of those with just a single protocol feature `StatelessValidation`. This may be a bit annoying so let's do it separately.